de/px-common.json

```diff
@@ -158,7 +158,11 @@
                     "title": "Online Interview",
                     "description": "Du kannst Zeitfenster ansehen und jenes auswählen, das für dich am günstigsten ist.",
                     "book": "Eine Session reservieren",
-                    "modalDescription": "In dieser Studie befindest du dich in einem Live-Videoanruf mit einem UX Forscher,  welcher dich um deine Gedanken und Meinung zu einer digitalen Erfahrung bitten wird. \n\nDiese Session wird ungefähr {{sessionDuration}} Minuten dauern. Du wirst auch wählen können, wann du teilnimmst, basierend auf den verfügbaren Zeitfenstern. "
+                    "modalDescription": "In dieser Studie befindest du dich in einem Live-Videoanruf mit einem UX Forscher,  welcher dich um deine Gedanken und Meinung zu einer digitalen Erfahrung bitten wird. \n\nDiese Session wird ungefähr {{sessionDuration}} Minuten dauern. Du wirst auch wählen können, wann du teilnimmst, basierend auf den verfügbaren Zeitfenstern. ",
+                    "mobileRequired": "",
+                    "smsReminder": "",
```

Comment:
expeting those to be empty?

Reply:
Not really, but the translations aren't available yet, apparently.